{"name": "Возврат ошибки авторизации несуществующего пользователя", "status": "passed", "description": "Проверка, что при попытки авторизоваться несуществующему пользователю, возвращается кода 404 и текст ошибки в теле ответа", "steps": [{"name": "Отправка POST-запроса для авторизации курьера", "status": "passed", "start": 1739026197715, "stop": 1739026197822}, {"name": "Проверка, что код ответа на создание заказа 404", "status": "passed", "start": 1739026197822, "stop": 1739026197822}, {"name": "Проверка, что в теле ответа содержится сообщение об ошибки", "status": "passed", "start": 1739026197822, "stop": 1739026197822}], "start": 1739026197715, "stop": 1739026197823, "uuid": "c8630b80-6ad7-42ea-8cb1-54575cb5e073", "historyId": "d970cf0f555024831bbe0248f62d6350", "testCaseId": "d970cf0f555024831bbe0248f62d6350", "fullName": "tests.test_auth_courier.TestAuthCourier#test_courier_login_nonexistent_user_returns_error", "labels": [{"name": "story", "value": "Логин курьера в системе"}, {"name": "feature", "value": "Работа с курьером"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_courier"}, {"name": "subSuite", "value": "TestAuthCourier"}, {"name": "host", "value": "bnovo-nb-0056"}, {"name": "thread", "value": "25328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_courier"}]}